cdap-ui/server/express.js

```diff
@@ -79,9 +78,39 @@ function getFaviconPath(uiThemeConfig) {
   return faviconPath;
 }
 
-function makeApp (authAddress, cdapConfig, uiSettings, uiThemeConfig) {
+function extractUITheme(cdapConfig) {
+  const uiThemePropertyName = 'ui.theme.file';
+  const DEFAULT_CONFIG = {};
+
+  if (!(uiThemePropertyName in cdapConfig)) {
+    log.info(`Unable to find ${uiThemePropertyName} property`);
+    return DEFAULT_CONFIG;
+  }
+
+  let uiThemeConfig = DEFAULT_CONFIG;
+  let uiThemePath = cdapConfig[uiThemePropertyName];
+  if (uiThemePath[0] !== '/') {
+    // if path doesn't start with /, then it's a relative path
+    // have to add the ellipses to navigate back to $CDAP_HOME, before
+    // going through the path
+    uiThemePath = `../../${uiThemePath}`;
+  }
+  try {
+    if (require.resolve(uiThemePath)) {
```

Comment:
If the `resolve` return `false`, the theme config silently fallback to default without any logging. I think it's better log  a warn, as it is not an expected behavior, right?